yarn/src/main/scala/org/apache/spark/scheduler/cluster/YarnClientSchedulerBackend.scala

```diff
@@ -125,6 +125,7 @@ private[spark] class YarnClientSchedulerBackend(
    */
   private def asyncMonitorApplication(): Unit = {
     assert(client != null && appId != null, "Application has not been submitted yet!")
+    val interval = conf.getLong("spark.yarn.client.progress.pollinterval", 1000)
```

Comment:
OK, is this superseded by #5305 ? it sounds like you have two different changes related to SPARK-3596 and I'm wondering whether they are mutually exclusive, or one or both is preferred, or what.